deployment/data-streams/changeset/jd_distribute_stream_jobs.go

```diff
@@ -21,6 +21,9 @@ var _ deployment.ChangeSetV2[CsDistributeStreamJobSpecsConfig] = CsDistributeStr
 type CsDistributeStreamJobSpecsConfig struct {
 	Filter  *jd.ListFilter
 	Streams []StreamSpecConfig
+
+	// NodePublicKeys specifies on which nodes to distribute the job specs.
+	NodePublicKeys []string
```

Comment:
Same comment as on LLO - We usually do this by node name, will we plan to do this mapping from CLD or why not just use node name to start with if it's a label?